enhance/fix clipboard

kevinslin opened the pull request:
- chore: publish patch
- fix(views): enable copy plaintext from preview

nickolay-kondratyev commented:
The change in packages/common-server/src/etc.ts  looks reasonable. 2 things though:

1. There are 33 files modified in this commit, could we decouple those seemengly unrelated changes from the change that is within packages/common-server/src/etc.ts 

2. Just manual validation in the PR comment: 
  2a. Does it work when copy pasting into Gmail(or gmail like client)?
  2b. Does it work when copy pasting into VSCode editor?

kevinslin commented:
Ah, 

> The change in packages/common-server/src/etc.ts looks reasonable. 2 things though:
> 
>     1. There are 33 files modified in this commit, could we decouple those seemengly unrelated changes from the change that is within packages/common-server/src/etc.ts
> 
>     2. Just manual validation in the PR comment:
>        2a. Does it work when copy pasting into Gmail(or gmail like client)?
>        2b. Does it work when copy pasting into VSCode editor?

Had non-rebased changes. Rebased and cleaned up. Only one file fix.

2a. Does it work when copy pasting into Gmail(or gmail like client)? yes
2b. Does it work when copy pasting into VSCode editor? yes
